challenger/ExecutionPoker.js

```diff
@@ -0,0 +1,387 @@
+'use strict';
```

Comment:
What changes did you incorporate against the class in `utils/ExecutionPoker.js` ?
If they are not specific, can we integrate those back in `utils/ExecutionPoker.js`,
so that we don't have to maintain basically two classes which should be doing the same thing. 🙂

Reply:
That was my plan, but it's not finished and i don't want to break something and then fix it until the stuff will be grilled enough :-) So i think i'll do that when fix dispute issues